specification/appcomplianceautomation/resource-manager/Microsoft.AppComplianceAutomation/preview/2022-11-16-preview/appcomplianceautomation.json

```diff
@@ -0,0 +1,1374 @@
+            "type": "string"
+          }
+        }
+      }
+    },
+    "ReportProperties": {
+      "type": "object",
+      "description": "Report's properties.",
+      "required": [
+        "timeZone",
+        "triggerTime",
+        "resources"
+      ],
+      "properties": {
+        "id": {
+          "type": "string",
+          "description": "Report id in database.",
+          "readOnly": true
+        },
+        "status": {
+          "description": "Report status.",
+          "enum": [
+            "Active",
+            "Failed",
+            "Disabled"
+          ],
+          "readOnly": true,
+          "type": "string",
+          "x-ms-enum": {
+            "name": "ReportStatus",
+            "modelAsString": true
+          }
+        },
+        "tenantId": {
+          "type": "string",
+          "description": "Report's tenant id.",
+          "readOnly": true
+        },
+        "reportName": {
+          "description": "Report name.",
+          "type": "string",
+          "readOnly": true
+        },
+        "offerGuid": {
+          "description": "Report offer Guid.",
+          "type": "string"
+        },
+        "timeZone": {
+          "type": "string",
+          "description": "Report collection trigger time's time zone."
+        },
+        "triggerTime": {
+          "format": "date-time",
+          "description": "Report collection trigger time.",
+          "type": "string"
+        },
+        "nextTriggerTime": {
+          "format": "date-time",
+          "type": "string",
+          "description": "Report next collection trigger time.",
+          "readOnly": true
+        },
+        "lastTriggerTime": {
+          "format": "date-time",
+          "type": "string",
+          "description": "Report last collection trigger time.",
+          "readOnly": true
+        },
+        "subscriptions": {
+          "type": "array",
+          "description": "List of subscription Ids.",
+          "items": {
+            "type": "string",
+            "description": "Single subscription Id."
+          },
+          "readOnly": true
+        },
+        "resources": {
+          "type": "array",
+          "description": "List of resource data.",
+          "items": {
+            "description": "Single resource metadata.",
+            "$ref": "#/definitions/ResourceMetadata"
+          },
+          "x-ms-identifiers": [
+            "resourceId"
+          ]
+        },
+        "complianceStatus": {
+          "readOnly": true,
+          "description": "Report compliance status.",
+          "$ref": "#/definitions/ReportComplianceStatus"
+        },
+        "provisioningState": {
+          "readOnly": true,
+          "description": "Azure lifecycle management",
+          "$ref": "#/definitions/ProvisioningState"
+        }
+      }
+    },
+    "ReportResource": {
+      "type": "object",
+      "description": "A class represent a AppComplianceAutomation report resource.",
```

Comment:
>a [](http://example.com/codeflow?start=40&length=1)

Should be 'an AppComplianceAutomation...', not 'a'

Reply:
Fixed.